### PR TITLE
Login NIH Fix:

### PR DESCRIPTION
**Pull request by ogwuazors**
- other idps disabled for now. can be removed later. will likely need a story to detail the removal of options.

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: src/bento/siteWideConfig.js [hunk 1]
```diff
@@ -5,4 +5,17 @@ export default {
   enableAuthentication: true,
   // List for options for authentication empty array defaults to google
   authEndPoint: ['google'], // authEndPoint: []
```
Comment (iksheth) [inline src/bento/siteWideConfig.js]: @ogwuazors Can you remove "authEndPoint" it's changed to "authProviders" now.

File: src/bento/userLoginData.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export const pageTitle = 'Login';
+
+export const loginProvidersData = {
+  google: {
+    key: 'google',
+    icon: 'https://raw.githubusercontent.com/CBIIT/datacommons-assets/main/bento/images/icons/png/google.png',
+    loginButtonText: 'Sign in with Google',
```
Comment (iksheth) [inline src/bento/userLoginData.js:7]: Can we remove "Sign in with" or "Sign in" from each button?

File: src/components/Auth/AuthProvider.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+import { useLazyQuery } from '@apollo/client';
+import { useGoogleLogin } from 'react-use-googlelogin';
+import env from '../../utils/env';
+import { signInRed, signOutRed } from './state/loginReducer';
+import { storeInLocalStorage, deleteFromLocalStorage } from '../../utils/localStorage';
+
+import GET_USER_DETAILS from '../../bento/authProviderData';
+
+const AUTH_API = env.REACT_APP_AUTH_SERVICE_API;
+const GOOGLE_CLIENT_ID = env.REACT_APP_GOOGLE_CLIENT_ID;
```
Comment (iksheth) [inline src/components/Auth/AuthProvider.js]: Can you add support for situation when "REACT_APP_GOOGLE_CLIENT_ID"  is undefined? 

For More info: https://github.com/CBIIT/bento-frontend/pull/672/commits/d94c384d4bde7d91078b15f5a8f3d12614856b89

File: src/bento/siteWideConfig.js (hunk 1 above)
Comment (iksheth) [inline src/bento/siteWideConfig.js:7]: @ogwuazors Can we remove "authEndPoint" property from here. It's no longer used.

**Review by iksheth — CHANGES_REQUESTED** [review]

File: src/components/Auth/AuthProvider.js (hunk 3 above)
Comment (ogwuazors) [reply src/components/Auth/AuthProvider.js]: Made the change for this.

File: src/bento/userLoginData.js (hunk 2 above)
Comment (ogwuazors) [reply src/bento/userLoginData.js:7]: Buttons should now only say the name of the IDP being used for logging in

**Review by iksheth — APPROVED** [review]
Looks good to me 👍🏻

**Pull request merged**